Makefile

```diff
@@ -26,10 +26,6 @@ export BIN_DIR=$(PROJ_ROOT)/bin
 export GTEST_DIR=$(PROJ_ROOT)/lib/googletest/googletest
 export PARSER_DIR=$(PROJ_ROOT)/lib/nginx-configparser
 
-.PHONY: deploy
-deploy: compile
-	cd deploy && bin/webserver base_config
-
```

Comment:
I agree that my `make deploy` is kinda useless here, but now that we have the script we should have a `make deploy` target that runs scripts/deploy.sh